identity.go

```
@@ -289,7 +343,10 @@ func (id *ID) SetCert(pemStr string) error {
 		id.Config.Cert = StoragePem + ":" + pemStr
 		return fmt.Errorf("could not save client certificate, location scheme not supported for saving (%s):\n%s", id.Config.Cert, pemStr)
 	case StorageFile, "":
-		f, err := os.OpenFile(id.Config.Cert, os.O_RDWR, 0664)
+
+		absPath, err := filepath.Abs(id.Config.Cert)
```

ignoring this err on purpose?

---

fixed